src/app/domains/MediaLibrary/combined/MediaLibraryModal/MediaLibraryModal.template.js

```diff
@@ -173,36 +194,85 @@ function MediaLibraryModal(props) {
               bg="#f6f9fe"
               className="custom-scroll">
               {/* Here should be list of data Images/Video */}
-              {formsList.map((item) => (
+              {imagesList.map((item) => (
                 <Box mr={3} mb={3}>
-                  <FormSimpleView
-                    key={item}
-                    title={item.title}
-                    subtitle={item.subtitle}
+                  <MediaLibraryItemSimpleView
+                    // key={item}
+                    // title={item.title}
+                    // subtitle={item.subtitle}
+                    // image={item.imageUrl}
+                    // name={item.name}
+                    {...item}
                   />
                 </Box>
               ))}
-              <Box
-                bg="#eceff5"
-                mr={3}
-                mb={3}
-                borderRadius="8px"
-                width="150px"
-                height="150px"
-                display="flex"
-                alignItems="center"
-                justifyContent="center"
-                style={globalStyles.cursorPointer}
-                onClick={onAddForm}>
-                <PlusOutlined />
-              </Box>
+              {/*==================================*/}
+              <Upload
+                showUploadList={false}
+                name="file"
+                customRequest={(data) => {
```

Comment:
Move this handler function outside of the render method, please

Reply:
done